notifier/operator/operator.py

```diff
@@ -54,19 +56,17 @@
     raise Exception('Environment variable SMTP_FROM must be set')
 if not smtp_host:
     raise Exception('Environment variable SMTP_HOST must be set')
-if not smtp_tls_cert and not smtp_tls_cert_file:
-    raise Exception('Environment variable SMTP_TLS_CERT or SMTP_TLS_CERT_FILE must be set')
-if not smtp_tls_key and not smtp_tls_key_file:
-    raise Exception('Environment variable SMTP_TLS_KEY or SMTP_TLS_KEY_FILE must be set')
 
 # Write cert and key to tmp file and load
 if smtp_tls_cert:
     smtp_tls_cert_fd, smtp_tls_cert_file = mkstemp()
     with open(smtp_tls_cert_fd, 'w') as f:
         f.write(f"{smtp_tls_cert}\n{smtp_tls_key}\n")
 
-tls_context = smtplib.ssl.SSLContext()
-tls_context.load_cert_chain(smtp_tls_cert_file, smtp_tls_key_file)
+tls_context = smtplib.ssl.create_default_context()
```

Comment:
The change from `smtplib.ssl.SSLContext()` to `smtplib.ssl.create_default_context()` has caused some trouble. It seems that we are currently not verifying the server-side TLS cert and this switches it to check the server cert. We'll need a way to pass in a CA cert to enable this.

Reply:
I successfully tested with server cert validation by adding:
```
smtp_tls_ca_cert = os.environ.get('SMTP_TLS_CA_CERT', None)
smtp_tls_ca_cert_file = os.environ.get('SMTP_TLS_CA_CERT_FILE', None)
```
and
```
tls_context = smtplib.ssl.create_default_context(
    cadata = smtp_tls_ca_cert,
    cafile = smtp_tls_ca_cert_file,
)
```

Reply:
Added cert validate as described. I also updated helm chart in order to pass TLS_CA_CERT but able to test it in "dry-run" mode without real TLS CA certificate.